DuckDuckGo/Subscription/SubscriptionManager+StandardConfiguration.swift

```diff
@@ -53,5 +54,15 @@ extension DefaultSubscriptionManager {
                       authEndpointService: authEndpointService,
                       subscriptionEnvironment: subscriptionEnvironment)
         }
+
+        accountManager.delegate = self
+    }
+}
+
+extension DefaultSubscriptionManager: AccountManagerKeychainAccessDelegate {
+
+    public func accountManagerKeychainAccessFailed(accessType: AccountKeychainAccessType, error: AccountKeychainAccessError) {
+        PixelKit.fire(PrivacyProErrorPixel.privacyProKeychainAccessError(accessType: accessType, accessError: error))
     }
```

Comment:
What do you think about setting `dailyAndCount` for the frequency so we get a clear picture at number of users affected vs. number of total events?